include/ceed.h

```diff
@@ -254,17 +259,26 @@ typedef enum {
 
 CEED_EXTERN const char *const CeedInterlaceModes[];
 
+/// Argument for CeedElemRestrictionCreateStrided that L-vector is in
+/// the Ceed backend's preferred layout. This argument should only be used
+/// with vectors created by a Ceed backend.
+/// @ingroup CeedElemRestriction
+CEED_EXTERN CeedInt CEED_STRIDES_BACKEND[3];
```

Comment:
This is a flag value, and thus can be `const`, right?

Reply:
Right! I'll fix it